include/crocoddyl/core/integrator/rk4.hpp

```diff
@@ -58,26 +55,33 @@ class IntegratedActionModelRK4Tpl : public ActionModelAbstractTpl<_Scalar> {
   virtual void print(std::ostream& os) const;
```

Comment:
We haven't implement this method (same for the Abstract and Euler integrators).

Reply:
Probably I don't get your comment. The method seems to be implemented in the hxx file. Also for Euler.

Reply:
Yes, you're right. Apologies for the mistake.

Note:
It is still missing the abstract class, but this is a general ongoing issue in Crocoddyl. So, let's ignore it (so we can merge this work asap)